### PR TITLE
literalInclude Hugo shortcode

### PR DESCRIPTION
**Pull request by gj**
TODO:

- [x] Figure out why JS icons aren't showing up on code snippets.
- [x] Update CI so that we're running doc tests against examples in `new-docs` instead of examples in `docs`.

### CODE REVIEW

File: new-docs/layouts/shortcodes/literalInclude.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
+{{ $dynPath := .Params.dynPath }}
```
Comment (dhatch) [inline new-docs/layouts/shortcodes/literalInclude.html:1]: Could you add some docs in here about how to use this? Similar to what was in your standup update this morning.

Comment (gj) [reply new-docs/layouts/shortcodes/literalInclude.html]: 8b9fd7b — can add more after vacation, but hopefully that's useful enough for now 😄

**Comment by samscott89** [comment]
> 
> TODO:
> 
>     * [ ]  Figure out why JS icons aren't showing up on code snippets.
> 


Add `js -> node-js` to the map here: https://github.com/osohq/oso/blob/main/new-docs/themes/oso-tailwind/layouts/partials/fontawesome.html#L8

Also `rs -> rust`.

**Comment by samscott89** [comment]
> 
> 
> TODO:
> 
>     * [x]  Figure out why JS icons aren't showing up on code snippets.

https://github.com/osohq/oso/pull/710/commits/973d638d3a18b0f5f5166aed251ac88b0df056e8

> 
>     * [x]  Update CI so that we're running doc tests against examples in `new-docs` instead of examples in `docs`.

https://github.com/osohq/oso/pull/710/commits/d8fe69a3ddeec14de54d1f833632f6c65ece37b6
Running now...

**Comment by leina05** [comment]
Screen Shot 2021-02-16 at 1 28 43 PM
This seems messed up, but is it just me?

**Comment by leina05** [comment]
The header is showing the actual file name, but we don't want that.
Screen Shot 2021-02-16 at 1 31 59 PM

**Comment by samscott89** [comment]
> 
> 
> Screen Shot 2021-02-16 at 1 28 43 PM
> This seems messed up, but is it just me?

Probably not. I couldn't find any Go examples before to check it. Go is the only one which doesn't have an actual icon. Perhaps we just drop it for now.

**Comment by samscott89** [comment]
> 
> 
> The header is showing the actual file name, but we don't want that.
> Screen Shot 2021-02-16 at 1 31 59 PM


Ah, we would need a title override then. That's a good point.

**Review by leina05 — APPROVED** [review]
Taking styling bugs as a follow-up before publishing docs.

**Pull request merged**